src/sync/app.hpp

```diff
@@ -58,6 +57,181 @@ class App {
         realm::util::Optional<uint64_t> default_request_timeout_ms;
     };
 
+    /**
+     * A struct representing a user API key as returned by the App server.
+    */
+    struct UserAPIKey {
```

Comment:
Maybe rename to UserAPIKeyReponse for clarity?

Reply:
Seems a bit redundant? Or are you trying to differentiate from the actual `string` key within the struct?

Reply:
I was mostly thinking about what to make of this name if I just saw it in code somewhere. It is a minor anyway.

Reply:
I see your point, since the actual key is nested within the structure.